src/components/movie/MovieDetails.jsx

```diff
@@ -0,0 +1,28 @@
+import { useNavigate, useParams } from "react-router-dom";
+import { GetMoviesBasedOnId, DeleteMovie } from "../../API/movies";
+
+const MovieDetails = () => {
+  const { id } = useParams();
+  const navigate = useNavigate();
+  const { movies: movie, isPending, error } = GetMoviesBasedOnId(id);
+  const handleDelete = async () => {
+    await DeleteMovie(movie.id).then((resp) => {
+      if (resp.ok) navigate("/");
```

Comment:
```suggestion
      resp?.ok && navigate("/");
```
if res.ok is undefined, it won't give an error because used ? for optional chaining. It is also known as the elvis operator.

Reply:
Updated. This is good thing to learn.